GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Agena/RO_Agena.cfg

```diff
@@ -0,0 +1,797 @@
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Lunar Orbiter Payload Separation Mechanism
+	@manufacturer = Jet Propulsion Laboratory
+	@description = A roughly 1.5m payload separation mechanism specifically designed for the Nimbus probe series, providing room for the experiments mounted to the bottom of the instrument torus.
+	@mass = 0.01
+}
+
+
+// GATV Materials Bay
+@PART[bluedog_GATV_MaterialsBay]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Agena Target Vehicle 1.5m Materials Bay
+	@manufacturer = McDonnell Aircraft
+	@mass = 0.05
+	@attachRules = 1,1,1,1,0
+}
+
+
+// MultiPayload Adapter
+@PART[bluedog_Agena_MultiPayloadAdapter]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Agena Multiple Payload Adapter
+	@manufacturer = Lockheed
+	@description = Multiple payload adapter for the Agena upper stage. Build your first payload, and then attach this to the fairing base below. Decouple the bottom once the top payload is separated.
+	@mass = 0.02
+	@attachRules = 1,1,1,1,0
+}
+
+
+// Retro Thrust Module
+@PART[bluedog_Agena_RetroThrustModule]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Agena Retro Thrust Module
+	@manufacturer = Lockheed
+	@attachRules = 1,1,1,1,0
+	@mass = 0.005
+	@maxTemp = 773.15
+	@skinMaxTemp = 873.15
+	@MODULE[ModuleEnginesFX]
+	{
+		@minThrust = 2
+		@maxThrust = 2
+		@heatProduction = 100
+		%ullage = False
+		%pressureFed = False
+		%ignitions = 1
+		@PROPELLANT
+		{
+			@name = Nitrogen
+			@ratio = 1.0
+		}
+		IGNITOR_RESOURCE
+		{
+			name = ElectricCharge
+			amount = 0.25
+		}
+		@atmosphereCurve
+		{
+			@key,0 = 0 180
+			@key,1 = 1 100
+		}
+	}
+}
+
+
+// ATDA Battery
+@PART[bluedog_ATDA_Battery]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Augmented Target Docking Adapter Battery Unit
+	@manufacturer = McDonnell Aircraft
+	@attachRules = 1,1,1,1,0
+	@mass = 0.02
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 25
+		basemass = -1
+		TANK
+		{
+			name = ElectricCharge
+			amount = 25000
+			maxAmount = 25000
+		}
+	}
+}
+
+
+// ATDA VHF Antenna
+@PART[bluedog_ATDA_VHFAntenna]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Augmented Target Docking Adapter VHF Antenna
+	@manufacturer = McDonnell Aircraft
+	@mass = 0.02
+	!MODULE[ModuleDataTransmitter],* {}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		referenceGain = 1.5
+		RFBand = VHF
+	}
+}
+
+
+// ATDA RCS
+@PART[bluedog_ATDA_RCS]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Augmented Target Docking Adapter RCS System
+	@attachRules = 1,1,1,1,0
+	@manufacturer = McDonnell Aircraft
+	@mass = 0.1
+	@maxTemp = 900
+	@skinMaxTemp = 2000
+	!RESOURCE,* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 30
+		basemass = -1
+	}
+	%useRcsConfig = RCSBlockTenth

```

Comment:
Too weak (and probably too light), this was a spare Gemini Reentry Control system, which was fitted with 25 lbf thrusters